README.md

```diff
@@ -1,11 +1,12 @@
 # 🆔 spdx
 
-[![Build Status](https://github.com/EmbarkStudios/spdx/workflows/CI/badge.svg)](https://github.com/EmbarkStudios/spdx/actions?workflow=CI)
+[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](https://embark.dev)
+[![Embark](https://img.shields.io/badge/discord-ark-%237289da.svg?logo=discord)](https://discord.gg/dAuKfZS)
 [![Crates.io](https://img.shields.io/crates/v/spdx.svg)](https://crates.io/crates/spdx)
 [![Docs](https://docs.rs/spdx/badge.svg)](https://docs.rs/spdx)
-[![SPDX Version](https://img.shields.io/badge/SPDX%20Version-3.8-blue.svg)](https://shields.io/)
-[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
-[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](http://embark.rs)
+![SPDX Version](https://img.shields.io/badge/SPDX%20Version-3.8-blue.svg)
```

Comment:
```suggestion
[![SPDX Version](https://img.shields.io/badge/SPDX%20Version-3.8-blue.svg)](https://spdx.org/licenses/)
```

Oops, noticed the SPDX badge didn't actually link to the SPDX license list :stuck_out_tongue: